### PR TITLE
Fixed compiler warnings

### PR DESCRIPTION
**Pull request by pollocktoci**
These changes are to fix several compiler warnings.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,4 @@ lib*.dylib
 .vs/
 /tags
 /host_tools.mwc*
+*.recipe
```
Comment (mitza-oci) [inline .gitignore:47]: Could this be in local configuration instead of in the repo?

**Comment by pollocktoci** [comment]
Most likely.  Is that a separate .gitignore file?

File: .gitignore (hunk 1 above)
Comment (mitza-oci) [reply .gitignore:47]: https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

File: tools/rtpsrelay/RelayHandler.h [hunk 2]
```diff
@@ -246,7 +246,7 @@ class VerticalHandler : public RelayHandler {
 
   void populate_address_map(AddressMap& address_map, const GuidSet& to);
 
-  virtual uint32_t local_active_participants() const override { return guid_addr_set_map_.size(); }
+  virtual uint32_t local_active_participants() const override { return (uint32_t)guid_addr_set_map_.size(); }
```
Comment (mitza-oci) [inline tools/rtpsrelay/RelayHandler.h:249]: static_cast?

**Review by mitza-oci — CHANGES_REQUESTED** [review]
There are merge conflicts due to concurrent changes in rtpsrelay

**Review by mitza-oci — APPROVED** [review]

**Pull request merged**